log/log.go

```diff
@@ -114,11 +133,17 @@ type writer interface {
 
 // bothWriter implements writer and writes to both syslog and stdout.
 type bothWriter struct {
+	sync.Mutex
 	*syslog.Writer
-	stdoutLevel int
+	*stdoutWriter
 	syslogLevel int
-	clk         clock.Clock
-	stdout      io.Writer
+}
+
+// stdoutWriter implements writer and writes just to stdout.
+type stdoutWriter struct {
```

Comment:
It feels a little weirdly asymmetric to leave the write-to-syslog functionality directly inside bothWriter, while splitting out the write-to-stdout functionality into its own type. But I guess if the idea here is that the write-to-syslog functionality will be deleted in the near future, maybe it's not worth the effort of doing the same refactoring to it now.

Reply:
Yeah, agreed. I was tempted to rewrite it for more symmetry but decided it was better to avoid making the PR bigger.